### ISSUE TITLE
Calendar widget in week view swipe issue - scrolls back to current week on selecting a date

### ISSUE THREAD

**Issue by supersailorcode** [FlutterFlow/flutterflow-issues#643]
### Has your issue been reported?

- [x] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I am facing an issue with using the calendar widget in week view.

When I swipe to the left (by swiping on the screen) to view future dates and tap on a particular date, the calendar scrolls automatically back to the current week view.

If I view future dates by clicking on the right facing arrow, and tap on a date, this issue (of automatically scrolling back to the current week) does not appear.

https://user-images.githubusercontent.com/128390577/226409506-a73234c1-96f9-4a11-8afa-93038857ca60.mp4

### Expected Behavior

If I swipe the calendar dates and then tap a particular date, the calendar should not scroll back automatically to the current week dates.

### Steps To Reproduce

1. Create a new page
2. Add the Calendar widget
3. Set the initial date to current time
4. Enable the week view toggle
5. Preview the app in test mode
6. Swipe to the left (by swiping on the screen) to view future dates and tap on a particular date, the calendar scrolls automatically back to the current week view.

### Bug Report Code

IT4sjPHl5YVIrsdc1c/+YMdB/GAXJVw2bec0scp8bxAbCIzrBJkAdvWkXFdWXu2iS39lfUSmmk4y3PeOj9+WUO4nG0uCY4c51pFqVQ3KRkGVQZOBOrjtSnZfO/pTf2qH5MK02CNCJrRfLkIhwF6mJO7JSg/BNqazImdISq/LZO4=

### Context

I want to display a list of records filtered by the date selected the calendar widget. The user should be able to swipe on the calendar widget to view future dates and then tap on a particular date. On tapping a particular date, I wish to show records filtered by that date.

But due to this issue, the user will not have an idea on the currently selected date as the calendar automatically scrolls back to the current week on selecting a date.

### Environment

```markdown
- FlutterFlow version: 3.7.3
- Platform: Wed and MacOS desktop
- Browser name and version: Version 111.0.5563.64 (Official Build) (x86_64)
- Operating system and version affected: macOS Monterey 12.6.3
```

**Comment by tanczosm** [comment]
Looks like the flutter_flow_calendar widget is missing:

`
onPageChanged: (fd) {
  focusedDay = fd;
},
`

![image](https://user-images.githubusercontent.com/2651588/227753671-fb4849a0-5e95-4ed9-8e5a-db72e46fafe2.png)

**Comment by organicnz** [comment]
Hi @supersailorcode. Thank you for submitting the issue :)

Confirmed. It scrolls backwards when selecting future dates and scrolls forwards when I select previous dates on an already selected date :)

Video: https://www.loom.com/share/89f31afdd47046e9b9999deada0753fe

**Comment by supersailorcode** [comment]
Thank you @organicnz for confirming. Please let me know if any other information is required from me. Hoping the big is fixed soon 😇

**Comment by organicnz** [comment]
Hey @supersailorcode. Thank you. The FlutterFlow team will review this already-triaged issue :)

**Comment by agreaves** [comment]
Hi @supersailorcode, thanks for the bug report and apologies for now updating when we had a fix! We've fixed this (as seen in #938, which is actually a duplicate of this one).

Sorry for the delay, and thanks again for reporting!
